core/src/main/java/org/frankframework/http/RestListenerServlet.java

```diff
@@ -69,11 +66,11 @@ protected void service(HttpServletRequest request, HttpServletResponse response)
 		String body = "";
 
 		if(restPath.contains("rest-public")) {
-			response.setHeader("Access-Control-Allow-Origin", CorsAllowOrigin);
+			response.setHeader("Access-Control-Allow-Origin", corsAllowOrigin);
```

Comment:
Perhaps you can replace this with a reference to `org.springframework.http.HttpHeaders#ACCESS_CONTROL_ALLOW_ORIGIN`. Same for `org.springframework.http.HttpHeaders#ACCESS_CONTROL_EXPOSE_HEADERS` below.

Reply:
Goed idee. Doe ik!